src/tools/perf/lib/libperf.c

```diff
@@ -256,14 +268,23 @@ static void ucx_perf_test_reset(ucx_perf_context_t *perf,
     perf->prev.bytes        = 0;
     perf->prev.iters        = 0;
     perf->timing_queue_head = 0;
-    perf->offset            = 0;
-    perf->allocator         = ucx_perf_mem_type_allocators[params->mem_type];
+
     for (i = 0; i < TIMING_QUEUE_SIZE; ++i) {
         perf->timing_queue[i] = 0;
     }
     ucx_perf_test_start_clock(perf);
 }
 
+static void ucx_perf_test_reset(ucx_perf_context_t *perf,
```

Comment:
i would rename it to ucx_perf_test_init(), to indicate it has to be used only once now (and not used again, e.g after warmup)

Reply:
Will do.